integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -319,76 +330,47 @@ void testDomainK8sEventsNonExistingCluster() {
    * up processing the domain resource.
    * Cleanup by patching the domain resource to a valid location and introspectVersion to bring up all servers again.
    */
-  @Order(4)
   @Test
   @DisplayName("Test domain events for failed/retried domain life cycle changes")
-  void testDomainK8SEventsFailed() {
-    V1Patch patch;
-    String patchStr;
-    Domain domain = assertDoesNotThrow(() -> getDomainCustomResource(domainUid, domainNamespace1));
-    String originalDomainHome = domain.getSpec().getDomainHome();
-
-    OffsetDateTime timestamp = now();
+  void testK8SEventsFailedLifeCycle() {
     try {
-      logger.info("Shutting down all servers in domain with serverStartPolicy : NEVER");
-      patchStr = "[{\"op\": \"replace\", \"path\": \"/spec/serverStartPolicy\", \"value\": \"NEVER\"}]";
-      patch = new V1Patch(patchStr);
-      assertTrue(patchDomainCustomResource(domainUid, domainNamespace1, patch, V1Patch.PATCH_FORMAT_JSON_PATCH),
-          "patchDomainCustomResource failed");
+      V1Patch patch;
+      String patchStr;
+      Domain domain = createDomain(domainNamespace5, domainUid, pvName5, pvcName5, "NEVER");
+      assertNotNull(domain, " Can't create domain resource");
+
+      String originalDomainHome = domain.getSpec().getDomainHome();
+
+      OffsetDateTime timestamp = now();
 
       logger.info("Checking if the admin server {0} is shutdown in namespace {1}",
-          adminServerPodName, domainNamespace1);
-      checkPodDoesNotExist(adminServerPodName, domainUid, domainNamespace1);
+              adminServerPodName, domainNamespace5);
```

Comment:
You have a lot of changed lines where the only change is that you moved the content over another tab stop? Why? If it's just the configuration of your editor, can you update its formatting so that it doesn't make as many unnecessary changes?